packages/cli/src/commands/env/add.ts

```diff
@@ -12,6 +12,12 @@ export default class EnvAdd extends AuthCommand {
       description: 'Indicate that the environment variable will be locked.',
       default: false,
     }),
+    secret: Flags.boolean({
+      char: 's',
+      description: 'Indicate that the environment variable will be secret.',
+      default: false,
+      exclusive: ['locked'],
```

Comment:
Do you need `exclusive` on only one of the flags, or should it be on the locked one too?

Reply:
On one flag is enough

Reply:
TIL, thanks!